### PR TITLE
chore: add npm version script

### PR DESCRIPTION
**Pull request by vchimev**

### CODE REVIEW

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

**Comment by SvetoslavTsenov** [comment]
test

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -76,3 +77,40 @@ It's our turn from there on! We will review the PR and discuss changes you might
 ## Where to Start
 
 If you want to contribute, but you are not sure where to start - look for [issues labeled `help wanted`](https://github.com/NativeScript/nativescript-angular/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).
+
+
+## Publishing new versions
+
+Instructions how to publish a new version for **NativeScript Core Team Members**.
```
Comment (sis0k0) [inline CONTRIBUTING.md]: `Maintainers` instead of `NativesScript Core Team members`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -76,3 +77,40 @@ It's our turn from there on! We will review the PR and discuss changes you might
...
+
+1. Execute `npm install` to install dependencies and prepare the package for publishing:
+```bash
+cd nativescript-angular
```
Comment (sis0k0) [inline CONTRIBUTING.md]: cd nativescript-angular/nativescript-angular

**Pull request merged**